src/modules/add-site/components/create-site-form.tsx

```diff
@@ -34,7 +33,7 @@ interface SiteFormErrorProps {
 	className?: string;
 }
 
-interface SiteFormProps {
+interface CreateSiteFormProps {
```

Comment:
@sejas, JFYI, you suggested to rename it previously in one of my PRs, but I merged it before noticing your suggestion. I had a personal note of your suggestion, to remember and address in some future PR where such change will suite, so addressing it here :)